src/ui/components/Rating/index.js

```diff
@@ -71,7 +71,7 @@ export class RatingBase extends React.Component<InternalProps, StateType> {
   // that surrounds the read-only set of stars.
   renderTitle = (
     rating: ?number,
-    readOnly: boolean,
+    readOnly: boolean | void,
```

Comment:
is this the same as `readOnly?: boolean`? above we have it that way ..so I'm just maybe not clear on the difference so just double checking : )

Reply:
`renderTitle` is called with positional arguments, like `renderTitle(3, true, 4)`. This kind of function is hard to maintain in JavaScript -- it's easy to mess up a call when you are refactoring code. We use Flow to catch mistakes.

If I were to define the `readOnly` parameter (positional argument 2) as `readOnly?` then Flow would allow that argument to be *omitted* from the call. I don't want that. However, I do want to allow the *value* of `readOnly` to be a literal `undefined`.

Reply:
ahh i see. cool, thanks for breaking it down : )